### ISSUE TITLE
twisted/web/howto/client.xhtml mentions non-existent "IEntityBodyProvider"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4319]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4319|
|Type|defect|
|Created|2010-02-25 21:34:03Z|

I think it means `IBodyProducer`.

Attachments:

* [client.xhtml-4319.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c15/c157a917672a6f9ec3669728168c6c85275c0bcf/47999e8303fe1d5ad1f8601bd15fcf3a63f385c1.patch) (1286 bytes) - added by aoverbury on 2010-04-26 13:40:34Z - Changes every instance of IEntityBodyProvider to IBodyProducer

<details><summary>Searchable metadata</summary>

```
trac-id__4319 4319
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation_easy documentation easy
time__1267133643000000 1267133643000000
changetime__1272294902000000 1272294902000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="aoverbury's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|aoverbury removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

Looks great.  Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [28867]) Apply http client doc fix correcting the name of an interface

Author: aoverbury
Reviewer: exarkun
Fixes: [#4319](https://github.com/twisted/twisted/issues/4319)

Change references to `IEntityBodyProvider` into references to `IBodyProducer`.
The former does not exist, the latter does.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
